companies/lijuan/bubble-tea.php

```diff
@@ -0,0 +1,12 @@
+<?php include 'header.php';?>
```

Comment:
all naming is allow spacing and please follow the naming convention as other teammates do. Otherwise again will increase your workload badly. including images naming.

Reply:
You can take a look at the header.php file, I just put all your code there, so I can reuse it.